models/Error.yaml

```diff
@@ -25,13 +25,19 @@ properties:
     description: |
       Code is a network-specific error code. If desired, this code
       can be equivalent to an HTTP status code.
+
+      This is considered the authoritative indicator of the cause
+      of the error.
     type: integer
     format: int32
     minimum: 0
     example: 12
   message:
```

Comment:
@matheusd We added the new `Error.Details` field in `v1.4.0` to capture different messages related to the context of the error but expect `Error.Message` to stay the same for all errors of a particular code.

We do not yet `assert` that this is true in the `rosetta-cli` yet, but will begin to do so shortly now that we have the `Error.Details` field.